src/components/Blog.jsx

```diff
@@ -66,10 +66,15 @@ export function Blog() {
           </h1>
           <div className="mt-6 space-y-6 font-display text-2xl tracking-tight text-secondary">
             <p>
-              Lorem ipsum dolor sit amet, consectetur adipisicing elit.
-              Quibusdam dicta odio temporibus sequi non, ducimus rerum dolor
-              maiores, error tempore dolorem explicabo quas aperiam accusamus
-              dolores aliquam iusto. Ratione, exercitationem.
+              Chegaram ao fim as Jornadas do Direito Informático! 
+              Foram três dias de novo conhecimento, tecnologias <i>cutting-edge</i>
```

Comment:
There's a space missing here :)

![image](https://user-images.githubusercontent.com/70779845/225330510-8fa4bf9a-79cf-4733-9b71-74bca8d2758e.png)

Try moving the <i>cutting-edge</i> to the middle of a line